admin-dev/themes/new-theme/js/pages/product/product-map.ts

```diff
@@ -133,7 +133,7 @@ export default {
       modifiedFieldClass: 'combination-value-changed',
       invalidClass: 'is-invalid',
       editionModeClass: 'edition-mode',
-      fieldInputs: `.combination-list-row :input:not(#${bulkCombinationSelectAllId}):not(.${isSelectedCombinationClass})`,
+      fieldInputs: `.combination-list-row :input:not(#${bulkCombinationSelectAllInPageId}):not(.${isSelectedCombinationClass})`,
```

Comment:
I believe you're gonna need to add the other "select all" checkboxes here in the `:not` conditions Because this selector is used to serialize the list form content in edition mode and we need to avoid submitting the checkboxes

I'm not sure if this cause a bug or if it was just me being thorough but better safe than sorry ^^